### ISSUE TITLE
"Sort by usage" ASC or DESC configurable

### ISSUE THREAD

**Issue by Morgy93** [8bitPit/Niagara-Issues#2522]
### Is your feature request related to a problem?

This is an improvement idea from https://github.com/8bitPit/Niagara-Issues/issues/2491

### Describe the solution you'd like

The apps on the homescreen are sortable by usage while it is only ASC by default.
So that the most used apps are at the top.
I'd love to have the most used apps at the bottom with DESC sorting, so that they are easier to reach with the thumbs.

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by Devareddy1998** [comment]
Yes, i too am looking for this feature, as well as automatic updating of the most used apps list of 6-8 apps in desc order
